torchmdnet/models/utils.py

```diff
@@ -227,6 +225,9 @@ def forward(self, pos, batch):
             edge_weight = torch.norm(edge_vec, dim=-1)
 
         lower_mask = edge_weight >= self.cutoff_lower
+        if self.loop and mask is not None:
+            # keep self loops even though they might be below the lower cutoff
+            lower_mask = lower_mask | ~mask
```

Comment:
Is this also part of the PR?

Reply:
It's something else, I'll move it into a separate PR.